packages/mobile/ios/Podfile.lock

```diff
@@ -888,4 +888,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 86c7e909196d7920299e68babc12fcaf4882d3e8
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.11.3
```

Comment:
Why cocoapods changes

Reply:
this might have been me originally actually. whenever i pod install with lerna, the version is set to 1.11.2. I think its supposed to be at 1.11.3

Reply:
@Kyle-Shanks yeah this is the result of me clearing out mobile node_modules and running npm i, and just seeing what happens, might help you if it keeps reverting you